webapp-django/crashstats/settings/base.py

```diff
@@ -439,6 +439,7 @@ def path(*dirs):
 
 STATICFILES_STORAGE = 'pipeline.storage.PipelineCachedStorage'
 
+LESS_PATH = '/tmp/crashstats-static' if LOCAL_DEV_ENV else '/app/webapp-django/static'
```

Comment:
Instead of hard-coding it, you can use the `STATIC_ROOT` variable. So you can do:

```
'LESS_ARGUMENTS': '--global-var="base-path=\'' + STATIC_ROOT + '/crashstats/css/base\'"',
```

Pretty sure that should work.

In the local dev environment, it's set to `/tmp/crashstats-static/` and in all other environments, it's set to `/app/webapp-django/static` (not sure if that has a `/` at the end or not).

Reply:
Ah, gotcha. For some reason, I thought the STATIC_ROOT pointed somewhere else entirely. I will correct!